package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nextjs-google-analytics",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

Comment:
I can take care of this, I'll release a new version once this is merged, no need to update it on this PR.

Reply:
Sure, thanks!